roles/network_plugin/calico/templates/calico.env.j2

```diff
@@ -4,6 +4,11 @@ ETCD_CERT_FILE="{{ calico_cert_dir }}/cert.crt"
 ETCD_KEY_FILE="{{ calico_cert_dir }}/key.pem"
 CALICO_IP="{{ip | default(ansible_default_ipv4.address) }}"
 CALICO_IP6=""
+{% if calico_network_backend is defined and calico_network_backend == 'none' %}
```

Comment:
sorry I meant CLUSTER_TYPE, thanks for the info on `CALICO_NETWORKING_BACKEND`, looks like it is bird by default if not set